src/withDirection.jsx

```diff
@@ -30,9 +30,11 @@ export default function withDirection(WrappedComponent) {
   class WithDirection extends React.Component {
     constructor(props, context) {
       super(props, context);
-      this.state = {
-        direction: context[CHANNEL] ? context[CHANNEL].getState() : defaultDirection,
-      };
+      let direction = context[CHANNEL] ? context[CHANNEL].getState() : defaultDirection;
+      if (props.direction) {
```

Comment:
if a prop is going to be added, a `.propTypes` entry *must* be added for it.

Reply:
I updated the way props are assigned at the end of this file to address this, now direction is always an optional prop for the HOC